bldsys/cmake/module/FindVulkan.cmake

```diff
@@ -233,6 +233,17 @@ if("FATAL_ERROR" IN_LIST Vulkan_FIND_COMPONENTS)
     list(REMOVE_ITEM Vulkan_FIND_COMPONENTS "FATAL_ERROR")
 endif()
 
+# FindVulkan only works correctly with the default CMAKE_FIND_FRAMEWORK
+# value: FIRST. If LAST it will find the macOS dylibs instead of the iOS
+# frameworks when IOS is true. If ALWAYS it will fail to find the macOS
+# dylibs. If NEVER it will fail to find the iOS frameworks. If frameworks
+# are ever included in the SDK for macOS, the search mechanism will need
+# revisiting.
```

Comment:
I'm not sure this is correct.  When I run VULKAN_SDK/ios/setup_env.sh the current pathing seems to be correct.  Both CI and locally in my mac, I'm seeing the build get created successfully with current setup.  Could you confirm the correction here is needed?